core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -67,38 +70,37 @@ public ServerIT()
     }
 
     @Test
-    public void testInstall()
+    public void testInstallUninstall()
+            throws Exception
     {
-        testInstall("21");
+        // Release names as in the https://api.adoptium.net/q/swagger-ui/#/Release%20Info/getReleaseNames
+        testInstall("jdk-21.0.2+13", "/usr/lib/jvm/temurin-21", "21");
+        testUninstall("jdk-21.0.2+13", "/usr/lib/jvm/temurin-21", "21");
     }
 
-    private void testInstall(String javaVersion)
+    private void testInstall(String temurinReleaseName, String javaHome, String expectedJavaVersion)
     {
         String rpm = "/" + new File(rpmHostPath).getName();
-        String command = "" +
-                // install required dependencies that are missing in UBI9-minimal
```

Comment:
We could add the comments to the command with `#`

Reply:
I didn't change that comment. Please review it here: https://github.com/trinodb/trino/pull/21190 (this is ready to be merged - doesn't change the current coverage/JDK)